bundle/ezpublish_legacy/ngadminui/design/ngadminui/templates/window_controls.tpl

```diff
@@ -1,8 +1,12 @@
 {* Window controls *}
+{def $default_tab = ezini( 'ViewSettings', 'DefaultTab', 'ngadminui.ini' )}
+{* If no DefaultTab is defined, display view tab by default *}
```

Comment:
Do we need this if it's coming from the INI file? Could just look up the INI setting directly in the definitions below.